apps/backoffice-v2/src/lib/blocks/variants/DefaultBlocks/DefaultBlocks.tsx

```diff
@@ -38,7 +37,7 @@ export const DefaultBlocks = () => {
               );
             })}
           </TabsList>
-          <ScrollArea orientation={'vertical'} className={'h-[73vh]'}>
+          <ScrollArea orientation={'vertical'} className={'h-[73vh] pe-4'}>
```

Comment:
Removed me-4 from BlockCell in favour of the pe-4 on the container of all blocks.